src/document.rs

```diff
@@ -84,9 +84,23 @@ fn print_type_def(type_def: &TypeDefinition) -> String {
             )
         }
         TypeKind::InputObject(input) => {
+            let directives = if !type_def.directives.is_empty() {
+                format!(
+                    " {} ",
+                    type_def
+                        .directives
```

Comment:
duplicate code. Serializing directives should be moved to a separate function and reused everywhere.